.github/ISSUE_TEMPLATE/30-impl-proposal.md

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+---
+name: Specification Change Proposal
+about: Request or propose an implementation that solves one or more use cases.
+title: ''
+labels: 'impl. proposal :gear:'
+assignees: ''
+
+---
+
+**Description**
+
+<!-- Describe the implementation proposal's functionality here -->
+
+
+**Use cases**
+
+<!-- Link at least one use case issue ID which would be solved by this implementation proposal -->
+
+* Fixes #xxxx
```

Comment:
We might want to add a tickbox in here somewhere to ask if this would be a breaking change.

Reply:
added